### PR TITLE
Cockpit 185

### PR DESCRIPTION
**Pull request by Gundersanne**
https://gundersanne.github.io/cockpit-project.github.io/blog/cockpit-185.html

### CODE REVIEW

**Comment by Gundersanne** [comment]
network
network-responsive-dialogs

kdump
screenshot from 2019-01-09 14-09-12

users
screenshot from 2019-01-09 13-08-41

**Comment by garrett** [comment]
Thanks! Working on the images & blog post now.

**Comment by garrett** [comment]
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-185.html

File: _posts/2019-01-09-cockpit-185.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 185.
+
+### Responsive dialogs
+
+Dialogs on the network page and kdump page are now responsive. As are the
+password dialogs on the users page.
```
Comment (martinpitt) [inline _posts/2019-01-09-cockpit-185.md]: How about "Dialogs on the Networking and KDump pages, as well as the password dialogs on the Accounts page are now responsive"? This avoids repetitions  and uses their actual menu entry names.

File: _posts/2019-01-09-cockpit-185.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+{:.side-by-side}
+![network bridge dialog screenshot](/images/network-bridge-dialog-responsive.png)
+![kdump crash dialog screenshot](/images/kdump-crash-dialog-responsive.png)
+![user password dialog screenshot](/images/users-password-dialog-responsive.png)
+
+### Docker: Include kubernetes containers in CPU/memory graphs
+
```
Comment (martinpitt) [inline _posts/2019-01-09-cockpit-185.md]: This looks a little empty. Perhaps shorten the title to "in graphs", and be a bit more verbose in a sentence? But I don't have an immediate good idea here either.

**Comment by garrett** [comment]
I didn't handle mobile well with the side-by-side flex. I should add that, but it can land outside of this PR. It's not urgent. (The images just show up pretty small in mobile as-is.)

Possibilities would be to switch to grid or to use media breakpoints. But that's 100% in the CSS and can come later.

**Pull request merged**